tests/functional/UpdateDBTest.php

```diff
@@ -70,7 +77,7 @@ public function testFailedUpdate($last_successful_update, $expected_status_repor
         $this->drush('php-script', ['updatedb_script'], ['script-path' => __DIR__ . '/resources']);
 
         // Force re-run of woot_update_8101().
-        $this->drush('php:eval', array('drupal_set_installed_schema_version("woot", ' . $last_successful_update . ')'), $options);
+        $this->drush('php:eval', array('\Drupal::service("update.update_hook_registry")->setInstalledVersion("woot", ' . $last_successful_update . ')'), $options);
```

Comment:
Does anyone know what's up with backslashes and Windows PHP? This line works in Circle, but fails in Appveyor.

Reply:
@greg-1-anderson just remove the leading `\` from `'\Drupal::service(...)`

Reply:
That might help. I think that the issue might be that backslashes are getting doubled up in a place where that's not necessary. But I'm in favor of avoidance at this stage. 😄